packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -138,6 +141,12 @@ export const RemoteServerFactory = {
         // This is the ORIGIN under which we are serving the page.
         // because it's an origin, it does not end with a slash.
         options.appOrigin = process.env.APP_ORIGIN = `${options.protocol}://${options.appHostname}`
+
+        if ('SSR_ALLOW_COOKIES' in process.env) {
+            // Toggle cookies being passed and set
+            options.ssrAllowCookies = process.env.SSR_ALLOW_COOKIES?.toLowerCase() === 'true'
```

Comment:
Nice guard. This is also the right typing here, because Lambda will pass us in a string.